### PR TITLE
feat(datasets): REST API bulk delete

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY
New REST API endpoint for bulk deleting datasets

Screenshot 2020-10-12 at 13 11 06

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset/datasets/api.py [hunk 1]
```diff
@@ -489,3 +494,62 @@ def related_objects(self, pk: int) -> Response:
...
+        item_ids = kwargs["rison"]
+        try:
+            BulkDeleteDatasetCommand(g.user, item_ids).run()
+            return self.response(
+                200,
+                message=ngettext(
+                    "Deleted %(num)d dataset",
+                    "Deleted %(num)d datasets",
+                    num=len(item_ids),
+                ),
```
Comment (villebro) [inline superset/datasets/api.py:548]: Nice, I'll have to start using this, too 👍

File: superset/datasets/commands/bulk_delete.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+
+class BulkDeleteDatasetCommand(BaseCommand):
+    def __init__(self, user: User, model_ids: List[int]):
+        self._actor = user
+        self._model_ids = model_ids
+        self._models: Optional[List[SqlaTable]] = None
+
+    def run(self) -> None:
+        self.validate()
```
Comment (villebro) [inline superset/datasets/commands/bulk_delete.py:45]: Curious, is this something that could be done in a decorator?

**Review by villebro — APPROVED** [review]
LGTM. Just one question that probably isn't relevant (just me thinking out lout).

File: superset/datasets/commands/bulk_delete.py (hunk 2 above)
Comment (dpgaspar) [reply superset/datasets/commands/bulk_delete.py:45]: yes, there's some repetition of this one

**Pull request merged**